vernac/indschemes.ml

```diff
@@ -147,23 +147,23 @@ let try_declare_scheme what f internal names kn =
         alarm what internal
           (strbrk "Required constant " ++ str s ++ str " undefined.")
     | DeclareUniv.AlreadyDeclared (kind, id) as exn ->
-      let msg = CErrors.print exn in
-      alarm what internal msg
+        let msg = CErrors.print exn in
+        alarm what internal msg
```

Comment:
I would not change the spacing here (up to line 166).  It's already legible and you've not made any substantive changes.

Reply:
hmm, I'm actually not sure how this happened. I don't remember seeing it in the diff. I must have pressed a reformatting shortcut sometime when messing around with this part of the code. I'll remove it.

Reply:
It must be either cosmic rays or gremlins, certainly not anything you did.

Reply:
My diff tool has a setting that ignores whitespace only changes.  I never use that setting.